portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,67 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+class submitComment{
+	public String name, content;
+	public submitComment(String name, String content){
+		this.name = name;
+		this.content = content;
+	}
+}
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Denzil!</h1>");
-  }
+	@Override
+	public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+		Query query = new Query("Comment").addSort("name", SortDirection.DESCENDING);
+
+		DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+		PreparedQuery results = datastore.prepare(query);
+
+		List<submitComment> commentList = new ArrayList<>();
+		for (Entity entity : results.asIterable()) {
+			String userName = (String) entity.getProperty("name");
+			String userContent = (String) entity.getProperty("content");
+			submitComment user = new submitComment(userName, userContent);
+			commentList.add(user);
+		}
+		Gson gson = new Gson();
+
+		response.setContentType("application/json;");
+		response.getWriter().println(gson.toJson(commentList));
+	}
+	@Override
```

Comment:
Let's add an empty line between these two methods to help break up the file visually.